File: abc/abc330/a.cpp
// https://atcoder.jp/contests/abc330/submissions/48004159

#include <bits/stdc++.h>
using namespace std;

int main() {
  int n, l;
  cin >> n >> l;

  vector<int> a(n);
  for(auto &z : a) cin >> z;

  int ans = 0;
  for(auto x : a) {
    if(x >= l) {
      ans++;
    }
  }

  cout << ans;
}

File: abc/abc330/a.rs
// https://atcoder.jp/contests/abc330/submissions/48061521

use proconio::input;

fn main() {
    input! {
        n: u16,
        l: u16,
        a: [u16; n]
    }

    let mut ans = 0;
    for x in a {
        if x >= l {
            ans += 1;
        }
    }

    print!("{}", ans);
}

